test/parallel/test-internal-modules.js

```diff
@@ -1,9 +1,10 @@
 'use strict';
-var common = require('../common');
-var path = require('path');
-var assert = require('assert');
+const common = require('../common');
+const path = require('path');
+const assert = require('assert');
 
-assert.throws(function() {
+assert.throws(function(err, cb) {
+  if (err) return cb(err);
   require('internal/freelist');
 });
```

Comment:
Sorry, the snippet explaining the task probably should have been a bit more precise on what to do here – This function won’t be called with `err` and `cb` arguments.

[`assert.throws`](https://nodejs.org/api/assert.html#assert_assert_throws_block_error_message) takes a second argument that can be a regexp and validates the error message against that (like [this](https://github.com/nodejs/node/blob/479822c45ef527386390227a1f8e086f6128d9e8/test/parallel/test-http-status-reason-invalid-chars.js#L13-L15)). Do you think you could update this PR with that?